code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -410,6 +410,37 @@
 	item_state = "kneesock"
 	species_fit = list(INSECT_SHAPED)
 
+/obj/item/clothing/shoes/kneesocks/attackby(obj/item/weapon/W, mob/user)
+	..()
+	if(istype(W, /obj/item/weapon/soap))
+		to_chat(user, "<span class='notice'>You place \the [W] into \the [src].</span>")
```

Comment:
This message is sent immediately before the progress bar begins, but it should be sent only in case it succeeds.

Reply:
Resolved in latest commit